### PR TITLE
Update meta descriptions in SDK docs v5 - Closes #828

### LINKED ISSUE #828: Update meta descriptions in SDK docs v5
Parent issue #738

### PR DESCRIPTION
**Pull request by chris529**
Closes #828

### CODE REVIEW

File: modules/ROOT/pages/architecture/communication-architecture.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 = Communication Architecture
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Communication Architecture covers the action & events interfaces, API client, invoking actions & application actions & events.
```
Comment (Tschakki) [inline modules/ROOT/pages/architecture/communication-architecture.adoc]:
```suggestion
:description: Covers the action & events interfaces and explains how the different components of the Lisk SDK communicate via channels.
```

File: modules/ROOT/pages/architecture/index.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = Architecture overview
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Framework architecture overview covers the on & off chain architecture, registering modules, configuration & the genesis block.
```
Comment (Tschakki) [inline modules/ROOT/pages/architecture/index.adoc]:
```suggestion
:description: Provides an overview of the Lisk SDK architecture, including on-chain, off-chain and the communication architecture.
```

File: modules/ROOT/pages/architecture/off-chain-architecture.adoc [hunk 3]
```diff
@@ -1,5 +1,6 @@
 = Off-chain architecture
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The off-chain architecture covers how to create a plug in, including custom, timesamp, and default plug ins.
```
Comment (Tschakki) [inline modules/ROOT/pages/architecture/off-chain-architecture.adoc]:
```suggestion
:description: Explains what are plugins and how to create and use them in a blockchain application.
```

File: modules/ROOT/pages/architecture/on-chain-architecture.adoc [hunk 4]
```diff
@@ -1,5 +1,6 @@
 = On-chain architecture
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The on-chain architecture covers modules, custom assests and examples, together with the SDK default modules.
```
Comment (Tschakki) [inline modules/ROOT/pages/architecture/on-chain-architecture.adoc]:
```suggestion
:description: Explains what are modules and how to create and use them in a blockchain application.
```

File: modules/ROOT/pages/references/lisk-framework/index.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 = Lisk Framework
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Framework overview provides a high-level synopsis of the Lisk Framework architecture, including its modules and components, how they communicate, and also how to change the default configuration.
+:description: The Lisk Framework overview covers its modules and components including the BaseModule, the BaseAsset, and the BasePlugin.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-framework/index.adoc]:
```suggestion
:description: Covers references of the Lisk Framework, like the BaseModule, the BaseAsset and the BasePlugin. 
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 6]
```diff
@@ -1,5 +1,6 @@
 = Hello World
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Hello World implementation, including the blockcahin & client application, setup, modules, components & transactions are covered here.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
:description: How to create a simple blockchain application including a react frontend.
```

File: modules/ROOT/pages/tutorials/index.adoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
 = Tutorials
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The SDK Tutorials overview provides a list of all available Lisk SDK tutorials in the Lisk documentation. It specifies the complexity, the estimated time required, and the main learning points of each tutorial.
+:description: The SDK Tutorials overview gives a summary of the Hello World app, including the complexity, estimated time required, and the content.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/index.adoc]:
```suggestion
:description: The SDK Tutorials overview gives a summary of all available tutorials, including the complexity, estimated time required, and the content.
```

Please use general descriptions of the page, so we don't need to update it too often int he future.

**Review by Tschakki — CHANGES_REQUESTED** [review]
A lot of the pages. you reviewed were pages that are not part of the SDKv5 docs yet, so the pages have not been updated yet. No point of updating the meta descriptions there.

File: modules/ROOT/pages/tutorials/index.adoc (hunk 7 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/index.adoc]: OK, good point.

File: modules/ROOT/pages/architecture/communication-architecture.adoc [hunk 8]
```diff
@@ -1,5 +1,6 @@
 = Communication Architecture
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Covers the action & events interfaces and explains how the different components of the Lisk SDK communicate via channels.
```
Comment (Tschakki) [inline modules/ROOT/pages/architecture/communication-architecture.adoc]:
```suggestion
:description: Introduces the Lisk communication architecture, which is based on IPC Unix Sockets and WebSocket.
```

action and events interfaces will be moved to rpc endpoints page as discussed

File: modules/ROOT/pages/guides/app-development/configuration.adoc [hunk 9]
```diff
@@ -1,6 +1,6 @@
 = Configure a blockchain application
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk SDK Configuration page describes how to configure a blockchain application developed with the Lisk SDK.
+:description: How to configure a blockchain application together with all the config options, constants and genesis block.
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/app-development/configuration.adoc]:
```suggestion
:description: How to configure a blockchain application together with the most important config options and the genesis block.
```

File: modules/ROOT/pages/references/lisk-elements/chain.adoc [hunk 10]
```diff
@@ -1,6 +1,6 @@
 = Chain
 Mona Bärenfänger <mona@lightcurve.io>
-:description: Technical references regarding the chain package of Lisk Elements.
+:description: Technical references regarding the chain package pf Lisk Elements including updates, state store, & data access.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/chain.adoc]:
```suggestion
:description: Technical references regarding the chain package pf Lisk Elements including state store & data access.
```

File: modules/ROOT/pages/references/lisk-elements/transactions.adoc [hunk 11]
```diff
@@ -1,6 +1,6 @@
 = Transactions
 Mona Bärenfänger <mona@lightcurve.io>
-:description: Technical references regarding the transactions packages of Lisk elements. This consists of usage examples, available options and example responses.
+:description: Technical references regarding the transactions packages consisting of usage examples, available options and example responses.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/transactions.adoc]:
```suggestion
:description: Technical references regarding the transactions package consisting of usage examples, available options and example responses.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/architecture/communication-architecture.adoc (hunk 8 above)
Comment (chris529) [reply modules/ROOT/pages/architecture/communication-architecture.adoc]: 👍

File: modules/ROOT/pages/references/lisk-elements/chain.adoc [hunk 12]
```diff
@@ -1,6 +1,6 @@
 = Chain
 Mona Bärenfänger <mona@lightcurve.io>
-:description: Technical references regarding the chain package of Lisk Elements.
+:description: Technical references regarding the chain package pf Lisk Elements including state store & data access.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/chain.adoc]:
```suggestion
:description: Technical references regarding the chain package of Lisk Elements including state store & data access.
```

File: modules/ROOT/pages/references/lisk-elements/client.adoc [hunk 13]
```diff
@@ -1,6 +1,6 @@
 = Client
 Mona Bärenfänger <mona@lightcurve.io>
-:description: Technical references regarding the clients packages of Lisk Elements. This consists of references to all included sub-packages and general usage instructions.
+:description: Technical references for the client packages of Lisk Elements, including all sub-packages and general usage instructions.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/client.adoc]:
```suggestion
:description: Technical references for the client package of Lisk Elements, including all sub-packages and general usage instructions.
```

File: modules/ROOT/pages/references/lisk-elements/transaction-pool.adoc [hunk 14]
```diff
@@ -1,5 +1,5 @@
 = Transaction pool
-:description: This section covers the installation, updates and constants for the Transaction pool library.
+:description: This section covers the installation, updates, constants, specifications and data structures for the Transaction pool library.
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/transaction-pool.adoc]:
```suggestion
:description: This section covers the installation, updates, constants, specifications and data structures for the transaction pool package of Lisk Elements.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Review by Tschakki — APPROVED** [review]

**Pull request merged**